packages/atomic/src/components/search/atomic-search-box-recent-queries/atomic-search-box-recent-queries.ts

```diff
@@ -181,14 +202,7 @@ export class AtomicSearchBoxRecentQueries {
     };
   }
 
-  public render() {
-    if (this.error) {
-      return (
-        <atomic-component-error
-          element={this.host}
-          error={this.error}
-        ></atomic-component-error>
-      );
-    }
+  render() {
```

Comment:
This is not equivalent; we should decorate the render fucntion with the errorGuard